### PR TITLE
update i18n

### PR DESCRIPTION
**Pull request by Diluka**
1. complete pt
2. add zh
3. use browser language as default
4. add language switch in settings

### CODE REVIEW

File: packages/ui/src/index.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const uiConfig = JSON.parse(
   document.getElementById('__UI_CONFIG__')?.textContent || '{}'
 ) as UIConfig;
 
-initI18n({ lng: uiConfig.locale?.lng || 'en-US', basePath }).then(() => {
+initI18n({ lng: uiConfig.locale?.lng || navigator.language || 'en-US', basePath }).then(() => {
```
Comment (felixmosh) [inline packages/ui/src/index.tsx:21]: This may break translations since we support only 3 languages

Comment (Diluka) [reply packages/ui/src/index.tsx:21]: https://github.com/felixmosh/bull-board/blob/8856fb5277f43b8bbe3be10289c54a3e9a99d180/packages/ui/src/services/i18n.ts#L22

What I understand about this configuration is that if the incoming language does not exist, en-US will be used instead, right?

Comment (felixmosh) [reply packages/ui/src/index.tsx:21]: Yeap, correct...

Comment (felixmosh) [inline packages/ui/src/index.tsx:21]: What will happen if you change the language from settings & refresh the page?
It looks like we are not using the settings value again

Comment (Diluka) [reply packages/ui/src/index.tsx:21]: add a commit to S/L lang settings

Comment (felixmosh) [inline packages/ui/src/index.tsx:21]: Is it possible to use the settings value first, then the global ui-config?

Comment (Diluka) [reply packages/ui/src/index.tsx:21]: tried, but can't use hooks here

Comment (felixmosh) [reply packages/ui/src/index.tsx:21]: How about using the store it self (I'm not sure if zustand supports that)

**Review by felixmosh — APPROVED** [review]

**Comment by felixmosh** [comment]
Thank you for this PR 🙏🏽

**Pull request merged**